app/controllers/draws_controller.rb

```diff
@@ -174,6 +174,7 @@ def calculate_metrics
   def calculate_suite_metrics
     @suite_sizes = SuiteSizesQuery.new(@draw.suites.available).call
     @suite_counts = @draw.suites.available.group(:size).count
+    @suite_counts.default = 0
```

Comment:
Oo awesome - does this mean we can rid of `zeroed_group_hash` and `empty_groups_hash`? Didn't realize this was a thing, very cool.